polkadot/runtime/rococo/src/impls.rs

```diff
@@ -100,6 +100,12 @@ where
 		// Do `check_out` accounting since the XCM Executor's `InitiateTeleport` doesn't support
 		// unpaid teleports.
 
+		// withdraw the asset from `who`
+		let who_origin =
+			Junction::AccountId32 { network: None, id: who.clone().into() }.into_location();
+		let _withdrawn = xcm_config::LocalAssetTransactor::withdraw_asset(&roc, &who_origin, None)
```

Comment:
Can we also `log::error!` the actual error before raising the new error? For better logging and debugging